### DISCUSSION TITLE
For Seeed nRF52 Boards Library | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#81] [Q&A]
# For Seeed nRF52 Boards Library | Seeed Studio Wiki

For Seeed nRF52 Boards Library

https://wiki.seeedstudio.com/XIAO-BLE-Sense-Bluetooth_Usage/

**Comment by FV215B2244P** [comment]
This and other examples doesnt work on xiao ble nrf52840, i tried most of them and still my phone cant see the board in both the nrf connect app and default bluetooth settings. And i cant find any helpful info on the internet.

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.

**Comment by ofc8** [comment]
can you transfer a file from xiao sense board, using the android mobile app?

**Comment by minhphat97** [comment]
hey guys, my mobile app nRF Connect can detect BLE from Xiao nRF52840 Sense successfully, but nRF Connect for Desktop can detect other BLE except my BLE Xiao. I am wondering if I have to any step to setup for nRF Connect for Desktop to detect Xiao BLE?
